src/wp-includes/blocks.php

```diff
@@ -153,8 +153,10 @@ function register_block_style_handle( $metadata, $field_name ) {
 
 	$style_handle = generate_block_asset_handle( $metadata['name'], $field_name );
 	$block_dir    = dirname( $metadata['file'] );
-	$style_file   = realpath( "$block_dir/$style_path" );
-	$version      = file_exists( $style_file ) ? filemtime( $style_file ) : false;
+	$style_file     = realpath( "$block_dir/$style_path" );
+	$has_style_file = false !== $style_file;
+	$version      = ! $is_core_block && $has_style_file ? filemtime( $style_file ) : false;
+	$style_uri    = ( $is_core_block && $style_file ) || ! $is_core_block ? $style_uri : false;
```

Comment:
```suggestion
	$style_file     = realpath( "$block_dir/$style_path" );
	$has_style_file = false !== $style_file;
	$version        = ! $is_core_block && $has_style_file ? filemtime( $style_file ) : false;
	$style_uri      = ( $is_core_block && $style_file ) || ! $is_core_block ? $style_uri : false;
```
Assignment alignment needed here to align all of the variable assignment operators (for phpcs).